.husky/pre-commit

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
I don't think this header is required any more because `husky init` does not add it automatically